README.md

```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/clearcontainers/runtime.svg?branch=master)](https://travis-ci.org/clearcontainers/runtime)
-[![Build Status](https://semaphoreci.com/api/v1/clearcontainers/runtime/branches/master/shields_badge.svg)](https://semaphoreci.com/clearcontainers/runtime)
+[![Build Status](http://cc-jenkins-ci.westus2.cloudapp.azure.com/job/clear-containers-runtime-azure-ubuntu-16-04/badge/icon)](http://cc-jenkins-ci.westus2.cloudapp.azure.com/job/clear-containers-runtime-azure-ubuntu-16-04/)
+[![Build Status](http://cc-jenkins-ci.westus2.cloudapp.azure.com/job/clear-containers-runtime-azure-ubuntu-17-04/badge/icon)](http://cc-jenkins-ci.westus2.cloudapp.azure.com/job/clear-containers-runtime-azure-ubuntu-17-04/)
 [![Go Report Card](https://goreportcard.com/badge/github.com/clearcontainers/runtime)](https://goreportcard.com/report/github.com/clearcontainers/runtime)
```

Comment:
I'm seeing an error in the build of this result

Reply:
what do you mean, which error ?

Reply:
The badge for the report card is not rendering when the document is built. And when clicked, the link doesn't appear to be working. The other badges are however rendering fine and links work.

Reply:
Oh yeah I see but this bug is not introduced by this PR since I can see the same behavior here https://github.com/clearcontainers/runtime on the main page. I think there is an issue with the goreportcard rendered, but nothing I can do about.

Reply:
That is a fine explanation to me :) will approve this PR